### PR TITLE
#11 - Update implemented

### LINKED ISSUE #11: Create/update Packages files
We need to create/update Packages.gz when deb package is added to the repository.  
Check the repo structure [here](https://www.debian.org/doc/manuals/repository-howto/repository-howto#id442666) and Packages format [here](https://wiki.debian.org/DebianRepository/Format#A.22Packages.22_Indices)

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #11 
Implemented update, added `UpdateSlice` to `DebianSlice`.

### CODE REVIEW

File: src/main/java/com/artipie/debian/http/DebianSlice.java [hunk 1]
```diff
@@ -71,6 +75,16 @@ public DebianSlice(final Storage storage, final Permissions perms,
...
                 ),
+                new RtRulePath(
+                    new RtRule.Any(
+                        new ByMethodsRule(RqMethod.PUT), new ByMethodsRule(RqMethod.POST)
+                    ),
+                    new BasicAuthSlice(
+                        new UpdateSlice(storage, reponame),
+                        users,
+                        new Permission.ByName(perms, Action.Standard.READ)
+                    )
```
Comment (genryxy) [inline src/main/java/com/artipie/debian/http/DebianSlice.java:86]: @olenagerasimova in this case user should have permission to write

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one comment

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, approve

**Pull request merged**